docs/backend-api/resources/tracking/apn_settings.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: /apn_settings
-description: /apn_settings
+title: APN settings of tracker
+description: APN settings of tracker
 ---
 
-## read(…)
+## Get APN settings of tracker
 
-Gets APN, user, password, operatorName by phone number
+### read(...)
+
+Gets APN, APN user, APN password, Operator Name for registered device by phone number
```

Comment:
"Gets the APN name/user/password and mobile operator for registered device by phone number."